pkg/controller/plan/adapter/ovirt/builder.go

```diff
@@ -748,6 +752,21 @@ func (r *Builder) PopulatorVolumes(vmRef ref.Ref, annotations map[string]string,
 	return
 }
 
+func (r *Builder) mapStorageDomainToStorageClass() (map[string]string, error) {
+	sdToStorageClass := make(map[string]string)
+	dsMapIn := r.Context.Map.Storage.Spec.Map
+	for i := range dsMapIn {
+		mapped := &dsMapIn[i]
+		ref := mapped.Source
```

Comment:
do we need this var?

Reply:
I've just sort of copy-pasted the code above, sure - there is a room for improvement there. done